gradle/shipkit.gradle

```diff
@@ -10,3 +10,9 @@ afterEvaluate {
     //TODO let's create a property for this setting in 'shipkit' extension
     tasks.updateReleaseNotes.publicationRepository = "https://plugins.gradle.org/plugin/org.shipkit.java"
 }
+
+apply plugin: 'org.shipkit.upgrade-downstream'
+
+upgradeDownstream{
+    repositories = ['mockito/shipkit-example']
```

Comment:
Sure, I wanted to start with example to check if it works as expected and then add other projects